Code/Framework/AzToolsFramework/Tests/Prefab/PrefabUndoEditEntityTests.cpp

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Prefab/PrefabTestFixture.h>
+
+#include <AzToolsFramework/Entity/EditorEntityHelpers.h>
+#include <AzToolsFramework/Prefab/Undo/PrefabUndo.h>
+#include <AzToolsFramework/Prefab/Undo/PrefabUndoEntityOverrides.h>
+#include <Prefab/PrefabTestComponent.h>
+
+namespace UnitTest
+{
+    using PrefabUndoEditEntityTests = PrefabTestFixture;
+
+    TEST_F(PrefabUndoEditEntityTests, PrefabUndoEditEntityTests_EditEntity)
```

Comment:
Google test library appends the class name to the test name internally. So, the final name it'll generate for this test is PrefabUndoEditEntityTests_PrefabUndoEditEntityTests_EditEntity. The 'PrefabUndoEditEntityTests_' prefix to these tests can be removed as having underscores can cause a few issues in the tests. See this comment more information on this: https://github.com/o3de/o3de/pull/13926#discussion_r1063789238

Reply:
GP. I'll also create a task to clean up test names like these.